pkg/demoinfocs/common/equipment.go

```diff
@@ -339,13 +357,30 @@ func (e *Equipment) AmmoInMagazine() int {
 		return -1
 	}
 
-	// need to subtract 1 as m_iClip1 is nrOfBullets + 1
-	return val.IntVal - 1
+	isSource1 := reflect.TypeOf(val.Any).Kind() == reflect.Int
+	if isSource1 {
+		// need to subtract 1 as m_iClip1 is nrOfBullets + 1
+		return val.Int() - 1
+	}
```

Comment:
```suggestion
	s1Ammo, isSource1 := val.Any.(int)
	if isSource1 {
		// need to subtract 1 as m_iClip1 is nrOfBullets + 1
		return s1Ammo - 1
	}
```

might be a bit nicer

Reply:
either way, long term @BestAwperEver's #393 or followup thereof can be used in the future

Reply:
Changed!